get_openstack_info_open_port/install.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+
```

Comment:
I believe this file should be removed... ?

Reply:
Pushed to soon, now updated with the rest of the script

Reply:
"Pushed to soon, now updated with the rest of the script", 
sorry I could not understand this line. @matteopozza commented on the importance of install.sh file. You can remove this file only when "setuptools-rust" goes to requirement.txt file. On the other hand "pip install --upgrade pip" is a user specific command which he can run when new pip available. If your plan is to put setuptools-rust in the requirements file, then it will be nice to add version as well. ie setuptools-rust == 1.2.3.4.

Reply:
@matteopozza commented on Jira that in a virtualenv this will solve "Errno 13] Permission denied:" problem which I have also faced. Why dont we add this osclient environment. 
```setuptools-rust                   0.12.1```

Reply:
@matteopozza has commented that inside a virtual environment, pip can install packages from requirements.txt file without any sudo/root privilege. I have also found this works for me. If you have plan to use virtual environment, 
please add the instruction on README.md that we need to install a virtual environment first and then install requiremnts there.

Reply:
@khabiruddin I'm marking this as unresolved since you are commenting on it.

Reply:
There seems to be something with this install.sh file because it doesn't show the content from the branch here in Github unless you go to the branch/file to see it. It should contain now the pip commands and the call to setup.py. My comment "Pushed to soon, now updated with the rest of the script" (had a typo, I meant "too soon"), was related to the fact that I pushed before saving the file, but if you fetch the branch "CCCP-3689-bis" you can see that the file is actually different from what this comment says now.

Reply:
We discussed that the oslo_log is not related to this tool, but related to the use of OpenStack CLI (Queens) after using this tool without changing virtual environments. Something out of the scope of this tool, the tools doesn't require oslo_log, but Openstack CLI (Queens) requires oslo_log, so to use the CLI you have to use a different virtual environment.

Reply:
What is the importance of install.sh? Can we put the installation of setuptools-rust in somewhere else?

Reply:
The reason for the install.sh is not keep simple the instructions and thinking about possible new steps. I don't think that a readme file should have a lot of code.